debian/control

```diff
@@ -387,3 +388,9 @@ Architecture: any
 Depends: ${misc:Depends},
   regolith-wm-config
 Description: Configurations to control lid close behaviour
+
+Package: regolith-sway-kbd-layout
```

Comment:
nit/opinion: I prefer not abbreviating words in human-exposed strings (unless there is some rt benefit), just for general readability.  But it also is obvious to me that `kbd` is keyboard.